colors/index.ts

```diff
@@ -1,6 +1,104 @@
-export * from './elevation';
-export * from './deepBlue';
-export * from './grayBlue';
-export * from './neon';
-export * from './orange';
-export * from './red';
+import * as elevation from './elevation';
+import * as deepBlue from './deepBlue';
+import * as grayBlue from './grayBlue';
+import * as neon from './neon';
+import * as orange from './orange';
+import * as red from './red';
+
+import {
+  blackA,
+  blue,
+  blueA,
+  blueDark,
+  blueDarkA,
+  gray,
+  grayA,
+  grayDark,
+  grayDarkA,
+  green,
+  greenA,
+  greenDark,
+  greenDarkA,
+  purple,
+  purpleA,
+  purpleDark,
+  purpleDarkA,
+  slate,
+  slateA,
+  slateDark,
+  slateDarkA,
+  whiteA,
+} from '@radix-ui/colors';
+
+const allColors = {
+  ...elevation,
+  ...deepBlue,
+  ...grayBlue,
+  ...neon,
+  ...orange,
+  ...red,
+};
+
+type LightColors = {
+  [key: string]: string;
+};
```

Comment:
As both Light and Dark have the same type, and this is not exported, we can have only one type for both, like `Colors` or `ColorMap`